src/server/lib/__tests__/headers.test.ts

```diff
@@ -108,7 +108,6 @@ describe('Content Security Policy headers', () => {
           'o14302.ingest.sentry.io',
         ].join(' '),
       );
-      expect(splitCSPHeader).toContain('block-all-mixed-content');
```

Comment:
Is this no longer useful or is it no longer being generated?

Reply:
its no longer being generated by helmet as it's been deprecated https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/block-all-mixed-content

Reply:
Thanks!